Shared/Extensions/JellyfinAPI/DynamicDayOfWeek.swift

```diff
@@ -0,0 +1,38 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Foundation
+import JellyfinAPI
+
+extension DynamicDayOfWeek {
+
+    var displayTitle: String {
+        switch self {
+        case .sunday:
+            DayOfWeek.sunday.displayTitle ?? self.rawValue
```

Comment:
On DayOfWeek, we are using this to get the day of the week instead of us localizing it: https://github.com/jellyfin/Swiftfin/pull/1255#discussion_r1797778181

Because of this, I am just using this to steal the localization from there. I think I could get away with `!`  instead of ?? rawValue but in case of failure I thought this would be less crash-y.